tools/acrn-crashlog/README.rst

```diff
@@ -0,0 +1,124 @@
+ACRN-Crashlog
+#############
+
+Introduction
+************
+
+The ``ACRN-Crashlog`` is a debug tool to collect any useful logs/information after each crash/event occurred on the platform, including hypervisor, Service OS(SOS) and Android as a Guest(AaaG). It provides abundant logs for developers to debug/analyze the crashes. And it provides a flexible way to configure which crash/event to be concerned with xml file.
+
+Building
+********
+
+Build dependencies
+==================
+
+The ``ACRN-Crashlog`` tool depends on the following libraries (build and runtime):
```

Comment:
Does the reader need to do something with this list of libraries?  Are they already installed, and if not, how does the reader install them?

Reply:
The GSG has instructions on how to install them (with the exception of telemetrics-client which is specific to Clearlinux)